ddcz/views/users.py

```diff
@@ -117,3 +122,34 @@ def change_skin(request):
         redirect_url = "/"
 
     return HttpResponseRedirect(redirect_url)
+
+
+@require_http_methods(["HEAD", "GET", "POST"])
+def awaiting(request):
+    registrations = UzivateleCekajici.objects.all().order_by("-date")
+    awaiting = registrations.count() is not 0
+
+    if request.method == "POST":
+        if request.POST.get("f") == "patronize":
+            try:
+                profile_id = request.user.userprofile.id
+                registration = UzivateleCekajici.objects.get(
+                    id=request.POST.get(DRAGON_EGG_NAME)
+                )
+                registration.patronize(profile_id)
+            except UzivateleCekajici.DoesNotExist:
+                pass  # log in the future
+            except:
+                pass  # log if user does not exists or not having permissions
+            return HttpResponseRedirect(reverse("ddcz:awaiting-registrations"))
+
+    return render(
+        request,
+        "users/awaiting-registrations.html",
+        {
+            "registrations": registrations,
+            "awaiting": awaiting,
+            "dragon_keeper": DRAGON_EGG_NAME,
+            "dragon_confirming_levels": DRAGON_EGG_BIRTH_CONFIRMER_LEVEL,
```

Comment:
I vote for `dragon_egg_confirmation_levels` — just `dragon` is a nameclash with the editor system, unfortunately (and makes me think whether not to check that